app/controllers/Application.java

```diff
@@ -200,7 +207,15 @@ private String validateParameters(HashMap<String, String> parameters, Request re
         return MSG_WRONG_DATE_PARAMETER_FORMAT;
       }
     }
-    return null;
+    return validateLanguageParam(language);
+  }
+
+  private static String validateLanguageParam(String value) {
+    List<String> availableLanguages = Arrays.stream(languages.get("available").toString().split(" ")).toList();
```

Comment:
I'm not fond of doing this on every method call because it costs extra processing and time to generate the list of languages. My preference would be to only do this once during startup and keep the available languages in memory (which has the downside of using several bytes more memory usage).